### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ManuelFay**

### CODE REVIEW

File: datasets/fquad/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+The text used for the contexts are from the curated list of French High-Quality Wikipedia [articles](https://fr.wikipedia.org/wiki/Cat%C3%A9gorie:Article_de_qualit%C3%A9).
+
+### Who are the annotators?
+
+Annotations (spans and questions) are written by students of the CentraleSupélec school of engineering.
+
+## Data characteristics
+
+The texts are movie reviews written by members of the [Allociné.fr](https://www.allocine.fr/) community for various films. The reviews were written between 2006 and 2020. Further information on the kinds of films included in the dataset has not been documented.
```
Comment (ManuelFay) [inline datasets/fquad/README.md]: Change this

File: datasets/fquad/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+language_creators:
+- wikipedia
+languages:
+- fr
+licenses:
+- CC BY-NC-SA 3.0
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
```
Comment (ManuelFay) [inline datasets/fquad/README.md]: Check this

**Review by ManuelFay — COMMENTED** [review]
Should be good, not sure about the size categories, I put 10K to 100K - to be checked.

**Comment by yjernite** [comment]
@ManuelFay thanks you so much for adding a dataset card, this is such a cool contribution!

This looks like it uses an old template for the card we've moved things around a bit and we have an app you should be using to get the tags and the structure of the Data Fields paragraph :) Would you mind moving your text to the newer format (we're also asking contributors to keep the full template structure, even if some sections still have [More Information Needed] for the time being)

Here's the link to the instructions:
https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Out of curiosity, what was your landing point for filling out the card? Did you follow the "Update on Github" when navigating the datasets? Trying to make the instructions as clear as possible :)

**Comment by ManuelFay** [comment]
@yjernite 

Perfect, I'll follow the instructions when I have a bit more time tomorrow ! I was actually browsing the new contributions after the dataset sprint and realized most of the "old" datasets were not tagged, so I just copied and pasted the readme from another dataset and was not aware there was precise instructions... Will fix !

BTW, amazing job with the retriBert work, I used the contrastive + in-batch negative quite a bit for various projects. Probably neither the time nor place to talk about that but I was curious as to why, in your original work, you prefered using a simple projection in the last layer to differentiate the question vs answer embedding, rather than allowing for dropout in the dense layer or even just to fine-tune 2 different embedders for question + answer ?

**Comment by yjernite** [comment]
Cool! Looking forward to the next version!

Quick answer for retriBERT is that I expected a simple projection to generalize better and more importantly only having to store the gradients for the proj means training with larger batches :) If you want to keep chatting about it, feel free to send me an email!

**Comment by lhoestq** [comment]
Hi @ManuelFay ! 
If you're still interested in completing the FQuAD dataset card, note that we've generated one that is pre-filled.
Therefore feel free to complete it with the content you already have in your README.md.
This would be awesome ! And thanks again for your contribution :)

**Comment by ManuelFay** [comment]
Yo @lhoestq , just not sure about the tag table at the top, I used @yjernite eli5 template so hope it's okay ! Also want to signal the streamlit app for dataset tagging has a weird behavior with the size categories when filling in the form. 

Thanks to you guys for doing that and sorry about the time it took, i completely forgot about it !

**Comment by ManuelFay** [comment]
Quick idea pitch, dynamic datasets:

I am typically thinking about @yjernite ELI5 dataset which contains Reddit data, which is static once it has been downloaded from the Pushshift dump for the first time. This has 2 potential limitations:
- The data does not update with time, which is good for benchmarking but it could be nice to have a dataset that updates with the latest infos and trends.
- In terms of data regulations, having downloaded user data from a website (which might be somewhat personal) at one point in time, and then storing it, might hinder the "right of deletion" set forth by the GDPR etc... 

A solution could be to have datasets that either automatically re download and process data from their source periodically, or even to have datasets which are not even stored on HF servers but are rather downloaded on the fly by the user, the `datasets` library being used in this case as a middle-man interface to automate and smooth things out (enabling to circumvent some GDPR restrictions regarding data storage).

Probably not the place to suggest this, sorry :)

File: datasets/fquad/README.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+language_creators:
+- crowdsourced
+languages:
+- fr-FR
+licenses:
+- cc-by-nc-sa-3.0
+multilinguality:
+- monolingual
+size_categories:
+-1K<n<10K
```
Comment (lhoestq) [inline datasets/fquad/README.md:15]:
```suggestion
- 1K<n<10K
```

File: datasets/fquad/README.md [hunk 4]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-
+The text used for the contexts are from the curated list of French High-Quality Wikipedia [articles](https://fr.wikipedia.org/wiki/Cat%C3%A9gorie:Article_de_qualit%C3%A9).
 ### Annotations
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+Annotations (spans and questions) are written by students of the CentraleSupélec school of engineering. 
+Wikipedia articles were scraped and Illuin used an internally-developped tool to help annotators ask questions and indicate the answer spans.
+ Annotators were given paragraph sized contexts and asked to generate 4/5 non-trivial questions about information in the context.
```
Comment (lhoestq) [inline datasets/fquad/README.md:141]:
```suggestion
Annotators were given paragraph sized contexts and asked to generate 4/5 non-trivial questions about information in the context.
```

File: datasets/fquad/README.md [hunk 5]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
+
+Dataset Split | Number of Articles in Split | Number of paragraphs in split | Number of questions in split
+--------------|------------------------------|--------------------------|-------------------------
+Train | 117 | 4921 | 20731
+Validation | 768 | 51.0% | 3188
+Test | 10 | 532 | 2189
+=======
 | name  |train|validation|
 |-------|----:|---------:|
 |default| 4921|       768|
```
Comment (lhoestq) [inline datasets/fquad/README.md:127]:
```suggestion
```

File: datasets/fquad/README.md [hunk 6]
```diff
@@ -30,26 +52,28 @@
 ## Dataset Description
 
 - **Homepage:** [https://fquad.illuin.tech/](https://fquad.illuin.tech/)
-- **Repository:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-- **Paper:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-- **Point of Contact:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Paper:**  [FQuAD: French Question Answering Dataset](https://arxiv.org/abs/2002.06071).
+- **Point of Contact:** [https://www.illuin.tech/contact/](https://www.illuin.tech/contact/])
```
Comment (lhoestq) [inline datasets/fquad/README.md:56]:
```suggestion
- **Paper:**  [FQuAD: French Question Answering Dataset](https://arxiv.org/abs/2002.06071)
- **Point of Contact:** [https://www.illuin.tech/contact/](https://www.illuin.tech/contact/)
```

File: datasets/fquad/README.md [hunk 7]
```diff
@@ -1,4 +1,26 @@
 ---
+annotations_creators:
+- crowdsourced
+extended:
+- original
+language_creators:
+- crowdsourced
```
Comment (lhoestq) [inline datasets/fquad/README.md:7]:
```suggestion
- crowdsourced
- found
```

File: datasets/fquad/README.md [hunk 8]
```diff
@@ -1,4 +1,26 @@
...
+languages:
+- fr-FR
```
Comment (lhoestq) [inline datasets/fquad/README.md:9]:
```suggestion
- fr
```

File: datasets/fquad/README.md [hunk 9]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
+No personal or sensitive information is included in this dataset. This has been manually verified by the dataset curators.
 
 ## Considerations for Using the Data
 
+Users should consider this dataset is sampled from Wikipedia data which might not be representative of all QA use cases.
+
 ### Social Impact of Dataset
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+ The social biases of this dataset have not yet been investigated.
```
Comment (lhoestq) [inline datasets/fquad/README.md:153]:
```suggestion
The social biases of this dataset have not yet been investigated.
```

File: datasets/fquad/README.md [hunk 10]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
 
 ### Discussion of Biases
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+ The social biases of this dataset have not yet been investigated, though articles have been selected by their quality and objectivity.
```
Comment (lhoestq) [inline datasets/fquad/README.md:157]:
```suggestion
The social biases of this dataset have not yet been investigated, though articles have been selected by their quality and objectivity.
```

File: datasets/fquad/README.md [hunk 11]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
 
 ### Other Known Limitations
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+ The limitations of the FQuAD dataset have not yet been investigated.
```
Comment (lhoestq) [inline datasets/fquad/README.md:161]:
```suggestion
The limitations of the FQuAD dataset have not yet been investigated.
```

File: datasets/fquad/README.md [hunk 12]
```diff
@@ -90,51 +114,61 @@ The data fields are the same among all splits.
...
 
 ### Dataset Curators
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+Illuin Technology:  [https://fquad.illuin.tech/](https://fquad.illuin.tech/)
 
 ### Licensing Information
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+ The FQuAD dataset is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/fr/) license.
```
Comment (lhoestq) [inline datasets/fquad/README.md:171]:
```suggestion
The FQuAD dataset is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/fr/) license.
```

**Review by lhoestq — APPROVED** [review]
Thanks you !

This looks all good :)
I just fixed minor formatting issues
